transform/snowflake-dbt/models/orgm/opportunity_snapshot.sql

```diff
@@ -52,7 +52,11 @@ WITH opportunity_totals AS (
         renewal,
         multi,
         renewal_multi,
-        monthly_billing
+        monthly_billing,
+        cpq_renewal_arr__c, 
```

Comment:
@sri-byte , are these fields documented anywhere? If not, can you open a bug on Jim to document them in the Confluence page, etc.? Thanks.

Reply:
Also, can you ask him to open a ticket on this request if he hasn't yet? Thanks.

Reply:
https://mattermost.atlassian.net/browse/MM-54120

Reply:
The request is more than just adding the fields - https://docs.google.com/document/d/1wwi7RXYFWqCjE_-vGMZFlKHv4T4-XapLJDW8nZGflKU/edit - are you/is he planning to add tickets for each request as well?

Reply:
The ticket for the charts (https://mattermost.atlassian.net/browse/MM-54125), I will link this in the Looker PR